src/pages/register.tsx

```diff
@@ -0,0 +1,211 @@
+import Link from 'next/link';
+import Image from 'next/image';
+
+const Register = () => {
+    const inputStyle =
+        'w-full bg-input-gray border-input-gray rounded-full p-2 px-3 mb-4';
+    return (
+        <div className="flex lg:justify-end">
+            <form className=" bg-white w-full lg:w-2/3 mt-40 lg:mt-0 rounded-t-[20px] lg:rounded-none lg:rounded-l-[20px] flex flex-col justify-start items-center text-purple font-ibm">
+                <h1 className="text-3xl font-bold mt-12 mb-2">ลงทะเบียน</h1>
+                <div className="flex flex-col justify-start items-center w-10/12 pt-6 lg:flex-row-reverse lg:justify-between">
+                    <div className="w-1/2 mb-6 lg:w-1/4">
+                        <div className="flex flex-row justify-between items-center">
+                            <div>อัปโหลดรูป</div>
+                            <button className="bg-[#E36130] flex flex-row justify-center items-center py-1 px-3 rounded-full">
```

Comment:
The tailwind.config.js file already has the color orange configured. Please use it instead of an arbitrary value.